src/components/form/skill/AddSkill.js

```diff
@@ -115,7 +115,7 @@ const AddSkill = ({ onClose, isEdit, values }) => {
       const skills = prevData['skills'];
 
       const index = skills.findIndex(skill => {
-        return skill.name === values.name;
+        return skill.id === values.id;
```

Comment:
```suggestion
const index = skills.findIndex(skill => skill.id === values.id)
```
We can remove the `return`

Reply:
done [database-integrate 4ff98e8] database-integrate:#7 - remove return from findIndex